### PR TITLE
Add document for warnings checks

### PR DESCRIPTION
**Pull request by XiongKezhi**

### CODE REVIEW

**Comment by XiongKezhi** [comment]
cc @timja

File: doc/Documentation.md [hunk 1]
```diff
@@ -908,6 +909,26 @@ Examples:
...
+If not disabled in the job configuration, this plugin will publish warnings to GitHub through [GitHub checks API](https://docs.github.com/en/rest/reference/checks).
+In the *Checks* view of each pull request ([example](https://github.com/jenkinsci/warnings-ng-plugin/pull/593/checks?check_run_id=1026691589)), issues statistics will be displayed.
+
+![checks](images/checks.png)
+
+When new issues are produced by a pull request, they will be added as annotations below the code where the issues are introduced.
+
+![check annotation](images/check-annotation.png)
+
+In order to disable the checks feature, set the property `skipPublishingChecks` to `true`, see the following example:
```
Comment (timja) [inline doc/Documentation.md:927]:
```suggestion
In order to disable the checks feature, set the property `skipPublishingChecks` to `true`:
```

**Review by timja — APPROVED** [review]
LGTM

**Review by uhafner — CHANGES_REQUESTED** [review]
Can you please add an additional sentence (and maybe image) that shows that additionally the status of the quality gate will be reflected in the status of the check?

**Review by uhafner — APPROVED** [review]
Thanks!

**Pull request merged**